readme.md

```diff
@@ -0,0 +1,130 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# House Frontend
```

Comment:
- [ ] Kindly add a screenshot of your frontend app to show it off.  Your repo will look a lot better and more professional. 💪